variables.tf

```diff
@@ -162,3 +162,8 @@ variable "vpc_name" {
   description = "The name of the VPC the infrastructure will be deployed to."
 }
 
+variable "key_name" {
+  default = ""
```

Comment:
Can you please set the default to `null`?

```HCL
variable "key_name" {
  default     = null
  description = "The ssh key to launch with the instances."
}

Reply:
Hey there, I will test it.
I put as "" because when I pulled the state it was actually an empty string instead of null.

Reply:
done